accord-core/src/main/java/accord/topology/Topologies.java

```diff
@@ -39,6 +39,8 @@
 //  (e.g. at least implementing Topologies by Topology)
 public interface Topologies extends TopologySorter
 {
+    enum SelectNodeOwnership { RESELECT, SHARE }
```

Comment:
could you add 1-line comments to both cases? They are obvious in context of the patch, but might be hard to understand after a while.

Also, maybe `RECOMPUTE` or `SLICE` or `RESLICE`?